lib/ncr_dispatcher.rb

```diff
@@ -0,0 +1,6 @@
+class NcrDispatcher < LinearDispatcher
```

Comment:
It's not the fact that there's a subclass, so much as I question whether that desire is actually NCR-specific/wide. Seems like it's either the case that it would be a requester-specific preference, or we should reduce the number of notifications for *all* requesters across the board. If this is just a hold-over until we know, though, totally fine.